app/controllers/sessions_controller.rb

```diff
@@ -8,7 +8,7 @@ def create
     if user && user.authenticate(params[:session][:password])
       session[:user_id] = user.id
       flash[:success] = "Successfully logged in!"
-      redirect_to user_admin_redirect
+      redirect_to check_cart
```

Comment:
some of this stuff has already been merged to dev_branch, it is only here because I manually merged my branch before uploading so I could make sure everything worked.